cirq-core/cirq/ops/phased_x_z_gate_test.py

```diff
@@ -199,6 +199,22 @@ def test_from_matrix_close_unitary(unitary: np.ndarray):
     )
 
 
+@pytest.mark.parametrize(
+    'unitary',
+    [
+        cirq.testing.random_unitary(2),
+        cirq.testing.random_unitary(2),
+        cirq.testing.random_unitary(2),
+        np.array([[0, 1], [1j, 0]]),
+    ],
+)
+def test_from_matrix_close_kraus(unitary: np.ndarray):
```

Comment:
Can we add another test that hits `cirq.unitary` as well ?

Reply:
There is already a test of `cirq.unitary`: https://github.com/quantumlib/Cirq/blob/master/cirq-core/cirq/ops/phased_x_z_gate_test.py#L196. This worked by decomposition even without a `_unitary_` implementation, but other protocols like `cirq.kraus` didn't.